man/fi_tcp.7.md

```diff
@@ -25,19 +25,38 @@ The following features are supported
 : *FI_EP_MSG* is the only supported endpoint type. Reliable
 datagram endpoint over TCP sockets can be achieved by layering RxM over
 tcp provider.
+: *FI_EP_RDM* is supported by layering ofi_rxm provider on top of the tcp provider.
 
 *Endpoint capabilities*
 : The tcp provider currently supports *FI_MSG*, *FI_RMA*
 
 *Progress*
 : Currently tcp provider supports only *FI_PROGRESS_MANUAL*
 
+*Shared Rx Context*
+The tcp provider supports shared receive context
+
+*Multi recv buffers*
+The tcp provider supports multi recv buffers
+
+
+# RUNTIME PARAMETERS
+
+The tcp provider check for the following enviroment variables -
+
+: *FI_TCP_IFACE* A specific can be requested with this variable
+; *FI_TCP_PORT_LOW_RANGE/FI_TCP_PORT_HIGH_RANGE* These variables are used to set the range of ports to be used by the tcp provider for its passive endpoint creation. This is useful where only a range of ports are allowed by firewall for tcp connections.
+
+
 # LIMITATIONS
 
 tcp provider is implemented over TCP sockets to emulate libfabric API. Hence
 the performance is lower than what an application might see implementing to
 sockets directly.
 
+# NOTES
+Enabling ADq support on ADq supported Intel NICs, would help achieve better performance.
```

Comment:
Please expand what ADq means.  If you have a link or description of search terms for how to determine if it exists and how to enable it, it would be helpful.

Reply:
I will add ADq related info later. Fixed other formatting issues.